src/.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
.idea folder should in in .gitignore file so that such files do not go to the pull request.